src/generated/cloud/gkemulticloud/v1/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code generated by sidekick. DO NOT EDIT.
+
+[package]
+name                 = "google-cloud-gkemulticloud-v1"
+version              = "0.2.0"
+description          = "Google Cloud Client Libraries for Rust - GKE Multi-Cloud API"
+edition.workspace    = true
+authors.workspace    = true
+license.workspace    = true
+repository.workspace = true
+keywords.workspace   = true
+categories.workspace = true
+
+[dependencies]
+async-trait = { version = "0.1" }
+bytes      = { version = "1", features = ["serde"] }
+gax        = { version = "0.2", path = "../../../../../src/gax", package = "gcp-sdk-gax", features = ["unstable-sdk-client"] }
```

Comment:
you know this, but you will have to rebase this or #1239 depending on the order of merging.

Reply:
yes, I need to rebase all open PRs.